steamworks_windows.go

```diff
@@ -3,6 +3,9 @@
 
 package steamworks
 
+// #include <stdlib.h>
+import "C"
```

Comment:
Don't use Cgo. Instead, we can create a C string pointer from a Go string by this:

```go
func goStringToCString(str string) (ptr unsafe.Pointer, free func()) {
    if !strings.HasSuffix(str, "\0") {
        str += "\0"
    }
    return unsafe.Pointer(&str[0]), func() { runtime.KeepAlive(str) }
}
```